doc/YAML/XS.swim

```diff
@@ -51,6 +51,22 @@ functions. Only `Load` and `Dump` are exported by default.
   the object is deleted. An example with File::Temp removing files can be found
   at [https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=862373]
 
+- $YAML::XS::ForbidDuplicateKeys` (since 0.84)
+
+  Default: false
+
+  The default might change to true at some point in the future.
```

Comment:
Let's not state this.

At some point I'd like to have all the config settings in a config object, and we can change the default at that time.

Reply:
done